### PR TITLE
eliminate EXPERIMENTAL_DA-gated code

### PR DESCRIPTION
**Pull request by LWinterberg**
Resolves: https://github.com/audacity/audacity/issues/5761

### CODE REVIEW

File: win/audacity.rc [hunk 1]
```diff
@@ -29,27 +25,15 @@ BEGIN
...
-            VALUE "InternalName", "DarkAudacity"
-            VALUE "LegalCopyright", "Copyright � 2018. All rights reserved."
-            VALUE "LegalTrademarks", "Audacity� is a registered trademark of Dominic Mazzoni."
-            VALUE "OriginalFilename", "DarkAudacity.exe"
-            VALUE "ProductName", "DarkAudacity"
-            VALUE "ProductVersion", AUDACITY_FILE_VERSION
-#else
             VALUE "CompanyName", "Audacity Team"
-            VALUE "FileDescription", "Audacity� Cross-Platform Sound Editor"
+            VALUE "FileDescription", "Audacity� Cross-Platform Sound Editor"
```
Comment (Paul-Licameli) [inline win/audacity.rc:29]: Looks like character encoding was messed up by your editor here

File: win/audacity.rc [hunk 2]
```diff
@@ -29,27 +25,15 @@ BEGIN
...
             VALUE "FileVersion", AUDACITY_FILE_VERSION
             VALUE "InternalName", "Audacity"
-            VALUE "LegalCopyright", "Copyright � 2018. All rights reserved."
-            VALUE "LegalTrademarks", "Audacity� is a registered trademark of Dominic Mazzoni."
+            VALUE "LegalCopyright", "Copyright � 2018. All rights reserved."
+            VALUE "LegalTrademarks", "Audacity� is a registered trademark of Dominic Mazzoni."
```
Comment (Paul-Licameli) [inline win/audacity.rc:33]: And here

File: libraries/lib-wx-init/HelpSystem.cpp [hunk 3]
```diff
@@ -232,14 +232,10 @@ void HelpSystem::ShowHelp(wxWindow *parent,
...
    wxASSERT(parent); // to justify safenew
-   wxString HelpMode = wxT("Local");
+   wxString HelpMode = wxT("Local"); //this probably always gets overwritten to FromInternet 
+                                     //TODO: remove code which handles local manual
 
-// DA: Default for DA is manual from internet.
-#ifdef EXPERIMENTAL_DA
    gPrefs->Read(wxT("/GUI/Help"), &HelpMode, {"FromInternet"} );
-#else
-   gPrefs->Read(wxT("/GUI/Help"), &HelpMode, {"Local"} );
```
[the remark this answers is not in the document]
Comment (LWinterberg) [reply libraries/lib-wx-init/HelpSystem.cpp:241]: This is intentional as we no longer do an offline manual as of https://github.com/audacity/audacity/commit/990d637816c59ed2ea4c6c443136c9aad90b4a68, so de-facto the dark audacity branch is the correct one here.

[the remark this answers is not in the document]
Comment (Paul-Licameli) [reply libraries/lib-wx-init/HelpSystem.cpp:241]: Then please say so in the amended commit comment.

**Review by Paul-Licameli — APPROVED** [review]

**Pull request merged**